app/projects/query.tsx

```diff
@@ -1,37 +1,83 @@
+import { FaDiscord, FaGithub } from 'react-icons/fa'
+import reactLogo from '~/images/react-logo.svg'
+import solidLogo from '~/images/solid-logo.svg'
+import vueLogo from '~/images/vue-logo.svg'
+import svelteLogo from '~/images/svelte-logo.svg'
+import angularLogo from '~/images/angular-logo.svg'
+import { useDocsConfig } from '~/utils/config'
+import { Link } from '@remix-run/react'
+import type { ConfigSchema, MenuItem } from '~/utils/config'
+
+export const repo = 'tanstack/query'
+
+export const latestBranch = 'main'
+export const latestVersion = 'v5'
+export const availableVersions = ['v5', 'v4', 'v3']
+
 export const gradientText =
   'inline-block text-transparent bg-clip-text bg-gradient-to-r from-red-500 to-amber-500'
 
-export const repo = 'tanstack/query'
+export const frameworks = {
+  react: { label: 'React', logo: reactLogo, value: 'react' },
+  solid: { label: 'Solid', logo: solidLogo, value: 'solid' },
+  vue: { label: 'Vue', logo: vueLogo, value: 'vue' },
+  svelte: { label: 'Svelte', logo: svelteLogo, value: 'svelte' },
+  angular: { label: 'Angular', logo: angularLogo, value: 'angular' },
+} as const
 
-const latestBranch = 'main'
+export type Framework = keyof typeof frameworks
 
-export const latestVersion = 'v5'
+export const createLogo = (version?: string) => (
+  <>
+    <Link to="/" className="font-light">
+      TanStack
+    </Link>
+    <Link to=".." className={`font-bold`}>
```

Comment:
Just to be consistent with code 3 lines above:
```suggestion
    <Link to=".." className="font-bold">
```

Reply:
Fixed - it does need another review, sorry!

I'll do this for the other PRs too